js/big-picture.js

```diff
@@ -0,0 +1,85 @@
+const COMMENTS_PER_PORTION = 5;
+
+const bigPicture = document.querySelector('.big-picture');
+const commentCount = document.querySelector('.social__comment-count');
+const commentList = document.querySelector('.social__comments');
+const commentsLoader = document.querySelector('.comments-loader');
+const body = document.querySelector('body');
+const cancelButton = document.querySelector('.big-picture__cancel');
+
+let commentsShown = 0;
+let comments = [];
+
+const createComment = ({ avatar, name, message }) => {
+  const comment = document.createElement('li');
+  comment.innerHTML =
+    '<img class="social__picture" src="" alt="" width="35" height="35"'>;
+    comment.classList.add('.social__comment');
```

Comment:
здесь в конце строки ковычку и скобку нужно местами поменять
```suggestion
    '<img class="social__picture" src="" alt="" width="35" height="35">';
    comment.classList.add('.social__comment');
```